src/osd/OSD.h

```diff
@@ -774,7 +774,7 @@ class OSDService {
 
 private:
   /// throttle promotion attempts
-  std::atomic_uint promote_probability_millis{1000}; ///< probability thousands. one word.
+  std::atomic<unsigned int> promote_probability_millis{1000}; ///< probability thousands. one word.
```

Comment:
what's the rationale behind this change?

Reply:
@shinobu-x ping?

Reply:
@chardan i don't feel strong either way. i just don't think we want to add code churn for no reason.

Reply:
@shinobu-x Also, the title might need rewording, since this change /stops/ using typedef-ed specializations, the opposite of what it suggests.